project.mml

```diff
@@ -1646,6 +1646,8 @@ Layer:
                                'laundry', 'dry_cleaning', 'beverages', 'perfumery', 'cosmetics', 'variety_store', 'wine', 'outdoor', 
                                'copyshop', 'sports', 'deli', 'tobacco', 'art', 'tea', 'coffee') THEN shop
                                ELSE 'other' END AS shop,
+            name,
```

Comment:
I added the name attribute (and other necessary ones for text rendering) to the amenity-points queries. At the same time I removed the now unnecessary parts of the text layer queries. Those are still needed, because there are other text features (e.g. water related) that also use the queries.